data/campaigns/Delfadors_Memoirs/scenarios/06_Swamps_of_Illuven.cfg

```diff
@@ -169,11 +210,222 @@
+                                [if]
+                                    [variable]
+                                        name=garrath_hostile
+                                        boolean_equals=no
+                                    [/variable]
+                                    [then]
+                                        # TODO: different dialogue if you've already given him an extra payment once
+                                        [message]
+                                            speaker=Garrath
+                                            message=_"Hey, that was one of OUR villages! Deal's off; we're no longer protecting you!"
+                                        [/message]
+                                        # Need to conditionalize here to ensure Garrath's reply makes sense:
+                                        [if]
+                                            [have_unit]
+                                                id=Harold
+                                            [/have_unit]
+                                            [then]
+                                                [message]
+                                                    speaker=Harold
+                                                    message=_"So... does that mean that OUR deal is back on now, Garrath?"
+                                                [/message]
+                                                [message]
+                                                    speaker=Garrath
+                                                    message=_"Fine, I guess..."
+                                                [/message]
+                                            [/then]
+                                        [/if]
+                                        [message]
+                                            speaker=Delfador
+                                            message=_"Wait, I'm sorry! I didn't realize you were so attached to that village! Please give me a chance to make it up to you!"
+                                        [/message]
+                                        [message]
+                                            speaker=Garrath
+                                            message=_"Well, I guess a little extra payment could persuade me to forgive you..."
+                                        [/message]
+                                        [store_gold]
+                                            variable=gold
+                                            side=1
+                                        [/store_gold]
+                                        [if]
+                                            [variable]
+                                                name=gold
+                                                less_than=20
+                                            [/variable]
+                                            [then]
+#ifdef DEBUG_MODE
```

Comment:
Better to not change dialogue based on whether debug mode is open or not.

Reply:
> Better to not change dialogue based on whether debug mode is open or not.

changed in d242e6a